tests/actions/globalPropertyEncounterType.test.js

```diff
@@ -0,0 +1,100 @@
+import configureMockStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+import moxios from 'moxios';
+import * as actions from '../../app/js/actions/globalPropertyEncounterType';
```

Comment:
Is it possible to import only the required modules using their names instead of importing all modules?